core/src/engine/ev/thread.cpp

```diff
@@ -187,6 +197,7 @@ void Thread::RunEvLoop() {
     AcquireImpl();
```

Comment:
While i'm at it: what's the purpose of this lock/unlock? - couldn't quite figure it out on my own